narayana/ArjunaJTA/jta/tests/classes/org/jboss/narayana/rts/TxnHelper.java

```diff
@@ -51,7 +50,6 @@ public static Set<Link> beginTxn(Client client, String txurl) throws IOException
                     Entity.entity(new Form(), MediaType.APPLICATION_FORM_URLENCODED_TYPE));
             Set<Link> links = response.getLinks();
 
-            EntityUtils.consume((HttpEntity) response.getEntity());
```

Comment:
I removed it because after updating the dependencies that line throws this runtime error: "java.lang.ClassCastException: class org.jboss.resteasy.specimpl.AbstractBuiltResponse$InputStreamWrapper cannot be cast to class org.apache.http.HttpEntity (org.jboss.resteasy.specimpl.AbstractBuiltResponse$InputStreamWrapper and org.apache.http.HttpEntity are in unnamed module of loader 'app')". Since I thought that line was not useful I removed it. If you think it is needed I will find out a way to make it work (I surely have to find a workaround)

Reply:
I think we just want to make sure the entity is fully read so instead of mixing resteasy with apache http I believe teh following should be good enough: `response.readEntity(String.class)`

Reply:
So if you do that then you can remove the dependency on org.apache.httpcomponents:

```
import org.apache.http.HttpEntity;
import org.apache.http.util.EntityUtils;
``

I'm not sure which other java files might need updating

Reply:
Thanks, I updated the file, the dependency is already removed. FYI the response.readEntity() takes no parameters.